src/frontend/mame/ui/slotopt.cpp

```diff
@@ -28,46 +30,9 @@ namespace {
 void *ITEMREF_RESET = ((void *)1);
 char DIVIDER[] = "------";
```

Comment:
While you're changing this - please make these `const` or `constexpr`.  Variable character arrays like this are a liability.